public/config.js

```diff
@@ -1,5 +1,5 @@
 CONFIG = {
-  mapper: 'SyntheaToV09', // mapper name from fhir-mapper -- allows mapping of incoming FHIR resources
+  mapper: null, // mapper name from fhir-mapper -- allows mapping of incoming FHIR resources
```

Comment:
Do we want to push this change @dehall?

Reply:
Yeah we do, there's no real benefit to using the mapper at the moment, and potential downside in that it can cause exceptions. I also changed it to null in my current PR.